guides/advanced_guides/ranking.md

```diff
@@ -18,13 +18,13 @@ Here is the list of all the `criteria` that are executed in this specific order
 
 ## Custom ranking rules
 
-Custom ranking rules gives you the possibility to add new rules. New rules can be added on attributes that has the `ranked` tag in the [schema](/main_concepts/indexes.md#ranked).
+Custom ranking rules gives you the possibility to add new rules. New rules can be added on attributes that has the `ranked` tag in the [schema](/guides/main_concepts/indexes.md#ranked).
 
 A rule is defined by an **attribute** and a **ascendent** `asc` or **descendent** `dsc` property.
 
 The name of the ranking rule is the name of the attribute on which the rule is made.
 
-For those rules to be applied by MeiliSearch on your search queries, it must be added to the [ranking order](/advanced_guides/ranking.md#ranking-order).
+For those rules to be applied by MeiliSearch on your search queries, it must be added to the [ranking order](/guides/advanced_guides/ranking.md#ranking-order).
```

Comment:
```suggestion
For those rules to be applied by MeiliSearch on your search queries, they must be added to the [ranking order](/guides/advanced_guides/ranking.md#ranking-order).
```
(Not sure of that because I'm not sure to really understand the sentence)